Kafka Connect/S2I status (#1748)
strimzi/strimzi-kafka-operator: documentation/book/appendix_crds.adoc

[options="header"]
|====
|Property       |Description

> |spec    1.2+<.<|The specification of the Kafka Connect deployment.

Reviewer: ```suggestion
|spec    1.2+<.<|The specification of the Kafka Connect cluster.
```

|xref:type-KafkaConnectSpec-{context}[`KafkaConnectSpec`]

> |status  1.2+<.<|The status of the Kafka Connect.

Reviewer: ```suggestion
|status  1.2+<.<|The status of the Kafka Connect cluster.
```

|string
|====

[id='type-KafkaConnectStatus-{context}']
### `KafkaConnectStatus` schema reference

Used in: xref:type-KafkaConnect-{context}[`KafkaConnect`]

[options="header"]
|====
|Property                   |Description
|conditions          1.2+<.<|List of status conditions.
|xref:type-Condition-{context}[`Condition`] array

> |observedGeneration  1.2+<.<|The generation of the CRD which was last reconciled by the operator.

Reviewer: ```suggestion
|observedGeneration  1.2+<.<|The generation of the CRD that was last reconciled by the operator.
```

|integer

> |restApiAddress      1.2+<.<|REST API address for managing and monitoring connectors.

Reviewer: ```suggestion
|restApiAddress      1.2+<.<|REST API endpoint for managing and monitoring Kafka Connect connectors.
```

|string
|====

[id='type-KafkaConnectS2I-{context}']
### `KafkaConnectS2I` schema reference

[options="header"]
|====
|Property       |Description

> |spec    1.2+<.<|The specification of the Kafka Connect deployment.

Reviewer: ```suggestion
|spec    1.2+<.<|The specification of the Kafka Connect Source-to-Image (S2I) cluster.
```

|string
|====

[id='type-KafkaConnectS2Istatus-{context}']
### `KafkaConnectS2Istatus` schema reference

Used in: xref:type-KafkaConnectS2I-{context}[`KafkaConnectS2I`]

[options="header"]
|====
|Property                   |Description
|conditions          1.2+<.<|List of status conditions.
|xref:type-Condition-{context}[`Condition`] array

> |observedGeneration  1.2+<.<|The generation of the CRD which was last reconciled by the operator.

Reviewer: ```suggestion
|observedGeneration  1.2+<.<|The generation of the CRD that was last reconciled by the operator.
```

|integer

> |restApiAddress      1.2+<.<|REST API address for managing and monitoring connectors.

Reviewer: ```suggestion
|restApiAddress      1.2+<.<|REST API endpoint for managing and monitoring Kafka Connect connectors with S2I support.
```

|string

> |buildConfigName     1.2+<.<|Name of the build configuration.

Reviewer: ```suggestion
|buildConfigName     1.2+<.<|The name of the build configuration.
```
